### PR TITLE
ui fixes

### PR DESCRIPTION
**Pull request by n-lina**

### CODE REVIEW

File: frontend/app/screens/welcome-screen/welcome-screen.tsx [hunk 1]
```diff
@@ -1,39 +1,77 @@
...
+  subheading: {
+    fontSize: 17,
+    textAlign: 'center',
+    color: lightseagreen, 
+    marginTop: spacing[3], 
+    fontStyle: 'italic'
+  },
+  image: {
+    width: windowWidth,
+    height: windowWidth*1.2,
```
Comment (jeremywiens) [inline frontend/app/screens/welcome-screen/welcome-screen.tsx:30]: This is too big on my phone, (I think mine is Pixel 2). Could you try this?

```suggestion
    height: windowHeight - 220,
```

File: frontend/app/screens/profile-screen/profile-screen.tsx [hunk 2]
```diff
@@ -102,6 +116,8 @@ export const ProfileScreen = observer(function ProfileScreen() {
   const totalLevelScore = userStore.getGoalsForNextLevel();
   const levelProgress = levelScore / totalLevelScore;
   // TODO: User userstore here
+  const myAwards = userStore.getAwards(false);
+  const myTitle = myAwards[myAwards.length-1].title;
```
Comment (jeremywiens) [inline frontend/app/screens/profile-screen/profile-screen.tsx:120]: This breaks if myAwards is empty.

Comment (jeremywiens) [reply frontend/app/screens/profile-screen/profile-screen.tsx:120]:
```suggestion
  const myTitle = (myAwards.length > 0) ? myAwards[myAwards.length-1].title : "Getting started...";
```

File: frontend/app/screens/welcome-screen/welcome-screen.tsx (hunk 1 above)
Comment (n-lina) [reply frontend/app/screens/welcome-screen/welcome-screen.tsx:30]: :o how is your screen height not a lot bigger than 1.2* the width?

File: frontend/app/screens/signin-screen/signin-screen.tsx [hunk 3]
```diff
@@ -121,9 +131,9 @@ export const SigninScreen = observer(function SigninScreen() {
                 .finally(() => setSigningIn(false))
             }
           />
-          <Text style={CONT_TEXT}>
+          {/* <Text style={CONT_TEXT}>
```
Comment (jeremywiens) [inline frontend/app/screens/signin-screen/signin-screen.tsx:134]: You didn't like this? 😱

File: frontend/app/screens/welcome-screen/welcome-screen.tsx (hunk 1 above)
Comment (n-lina) [reply frontend/app/screens/welcome-screen/welcome-screen.tsx]: fixed to make it a square

File: frontend/app/screens/signin-screen/signin-screen.tsx (hunk 3 above)
Comment (n-lina) [reply frontend/app/screens/signin-screen/signin-screen.tsx:134]: lol we can add it back !

**Comment by n-lina** [comment]
home screen has bugs when there are multiple times on the same day per stg. add goal test might not work anymore either. 
Steps to reproduce home screen bug: 
1. Add a new goal 
2. Add a stg with 3 different times all on Friday 
3. Try what happens when theres a few times on a different day as well. 
4. Go to home screen and do not refresh yet - i think it is correct at this point 
5. Refresh home screen - observe all the duplicates have the same time

File: frontend/app/screens/signin-screen/signin-screen.tsx (hunk 3 above)
Comment (n-lina) [reply frontend/app/screens/signin-screen/signin-screen.tsx]: added it back, its a nice touch

Comment (jeremywiens) [reply frontend/app/screens/signin-screen/signin-screen.tsx]: ahaha I appreciate it, at least I have some contribution to the screen... 😆

**Review by jeremywiens — APPROVED** [review]

**Pull request merged**